documentation/operations-manual.md

```diff
@@ -1 +1,36 @@
-# Travelcy Operations Manual
\ No newline at end of file
+# Travelcy Operations Manual
+
+## Overview
+Travelcy is a native android application developed in Kotlin with Android Studio.
+
+## Coding conventions
+We have been following Kotlins' standard coding convention, which can be found here https://kotlinlang.org/docs/reference/coding-conventions.html
+
+## Testing
+Tests are defined under seperate packages com.travelcy.travelcy (test) and com.travelcy.travelcy (androidTest), under which we define our tests, mocks of repositories/services etc.
+Creating tests is a necessary part of our workflow, and it is taken into consideration when reviewing pull requests.
+
+## Setting up the development environment
+1. Download and install Android Studio https://developer.android.com/studio/
+2. Set up git on your machine if you haven't already and clone the repository from here https://github.com/Travelcy/Travelcy
+3. You may have to download and install the most recent java sdk https://www.oracle.com/java/technologies/javase-jdk13-downloads.html
+
+## Pull request/review policy
```

Comment:
Would be good to say that pull request merging is blocked until the test run has been successful and at least one review has been approved.